ee/dtr/admin/install/index.md

```diff
@@ -64,14 +64,14 @@ DTR and UCP must not be installed on the same node, due to the potential for res
 
     * To install a specific version of DTR, replace `{{ page.dtr_version }}` with the  desired version in the [installation command](#step-3-install-dtr). (All DTR versions can be found on the [DTR release notes](/ee/dtr/release-notes/) page.)
 
-    * As DTR is deployed with self-signed certificates by default, UCP might not be able to pull images it. To automatically configure UCP to trust DTR, use the `--dtr-external-url <dtr-domain>:<port>` optional flag during installation, or during a reconfiguration.
+    * As DTR is deployed with self-signed certificates by default, UCP might not be able to pull images from it. To automatically configure UCP to trust DTR, use the `--dtr-external-url <dtr-domain>:<port>` optional flag during installation, or during a reconfiguration.
```

Comment:
Can you remove the "As", so something like "By default DTR uses self-signed certificates that may cause errors when UCP pulls images from it. To configure UCP to accept DTR self-signed certificates, use the '--dtr-external-url <dtr-domain>:<port>' flag when you install or configure UCP. "
I would also add a link to the reference topic for the flag

Reply:
Like your edit, @Dawn-Wood, and will (mostly) use it whole.